system/jlib/jcomp.cpp

```diff
@@ -953,7 +953,7 @@ class CCompilerWorker : implements IPooledThread, public CInterface
             e->errorMessage(sb);
             error.setown(e);
             if (sb.length())
-                PrintLog("%s", sb.str());
+                IERRLOG("%s", sb.str());
```

Comment:
Shouldn't the above lines be a replaced with something more concise like EXCLOG(e) ?